api/organisations/models.py

```diff
@@ -99,6 +99,19 @@ class Subscription(models.Model):
     cancellation_date = models.DateTimeField(blank=True, null=True)
     customer_id = models.CharField(max_length=100, blank=True, null=True)
 
+    CHARGEBEE = "ChargeBee"
```

Comment:
I prefer these to be e.g. `CHARGEBEE = "CHARGEBEE"` as that is what is stored at the database level. Then you know you don't need to worry about casing.